### PR TITLE
Add NEWS.md, a human-readable list of changes

### PR DESCRIPTION
**Pull request by malept**
As mentioned in https://github.com/maxogden/electron-packager/issues/235#issuecomment-170384955 and https://github.com/maxogden/electron-packager/pull/223#issuecomment-183989800.

Only contains changes from 5.2.0 onward. If someone else wants to backfill, feel free :grin:

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Changes by Version
+
+## [Unreleased]
+
+### Added
+
+* Add `tmpdir` parameter to specify a custom temp directory (#230)
```
Comment (malept) [inline NEWS.md]: Opinions on whether to actually link the PRs? I kind of wish GitHub would automatically do so.

Comment (kfranqueiro) [inline NEWS.md]: If we were to use GitHub's releases feature instead of add a file to the repo ourselves, it automatically links issues there.

Comment (malept) [inline NEWS.md]: I propose we do both. This way the person tagging the release is not on the hook to gather the release notes, it should be gathered as changes are added.

Comment (kfranqueiro) [inline NEWS.md]: FWIW, any contributor is able to create or edit a release draft on github at any time to keep track of release notes ahead of an actual release; e.g. with dgrid I try to keep a draft updated as each thing gets merged. Just have to be careful not to prematurely click the big green button on the page (why do they have to always have tempting big green buttons?!)

Not sure if that influences your thought of doing both a NEWS.md and github release notes.

Comment (malept) [inline NEWS.md]: > Just have to be careful not to prematurely click the big green button on the page (why do they have to always have tempting big green buttons?!)

Yeah, today I already accidentally clicked the "close PR" button.

The nice thing about the `NEWS.md` approach is that a user can tell what's unreleased - using just GitHub Releases doesn't really help with that use case. I have needed to figure out if feature X has been released or not when responding to issues.

**Comment by malept** [comment]
I added GitHub compare links to the version names, but for whatever reason, the diffs aren't quite right.

**Pull request merged**